twittermap/public/javascripts/map/controllers.js

```diff
@@ -1,5 +1,27 @@
 angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
   .controller('MapCtrl', function($scope, $window, $http, $compile, cloudberry, leafletData, cloudberryConfig) {
+
+    // add alert of IE
+    if (!L.Browser.ie) {
+    ieAlert();
```

Comment:
since it is a special function, can we just put the implementation here? the `alert` shouldn't be used anywhere else?